cli/src/main/java/com/box/l10n/mojito/cli/command/checks/CliCheckerExecutor.java

```diff
@@ -2,19 +2,30 @@
 
 import com.box.l10n.mojito.cli.command.extraction.AssetExtractionDiff;
 import java.util.List;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 public class CliCheckerExecutor {
 
   private final List<AbstractCliChecker> cliCheckerList;
 
-  public CliCheckerExecutor(List<AbstractCliChecker> cliCheckerList) {
+  private final Pattern excludedFilesPattern;
+
+  public CliCheckerExecutor(List<AbstractCliChecker> cliCheckerList, Pattern excludedFilesPattern) {
     this.cliCheckerList = cliCheckerList;
+    this.excludedFilesPattern = excludedFilesPattern;
   }
 
   public List<CliCheckResult> executeChecks(List<AssetExtractionDiff> assetExtractionDiffs) {
     return cliCheckerList.stream()
-        .map(check -> check.run(assetExtractionDiffs))
+        .map(
+            check -> {
+              if (check.getCliCheckerType() == CliCheckerType.CONTEXT_COMMENT_CHECKER) {
+                ((ContextAndCommentCliChecker) check)
+                    .setExcludedFilesPattern(this.excludedFilesPattern);
```

Comment:
Rather than do this here you can include the pattern in checker options and then retrieve it in the check itself.

Reply:
Done